e2e/cypress/tests/support/ui/channel.js

```diff
@@ -28,7 +28,7 @@ Cypress.Commands.add('uiCreateChannel', ({
     if (createBoard) {
         cy.get('#add-board-to-channel').should('be.visible');
         cy.findByTestId('add-board-to-channel-check').then((el) => {
-            if (!el.hasClass('checked')) {
+            if (el && !el.hasClass('checked')) {
```

Comment:
not part of this change, but while reviewing the code I noticed that it was worth also adding this validation to the e2e test